### PR TITLE
publiccoud: Strip all color codes from az output

### PR DESCRIPTION
**Pull request by cfconrad**
- Verification run: https://openqa.suse.de/t3781844 - publiccloud_ltp_syscalls@cfconrad/os-autoinst-distri-opensuse#fix/pc_strip_colors => running

### CODE REVIEW

File: lib/publiccloud/azure.pm [hunk 1]
```diff
@@ -24,11 +25,17 @@ has storage_account => 'openqa';
...
 has lease_id        => undef;
 
-# due to https://github.com/Azure/azure-cli/issues/9903 we need to trim 4 last chars in az output
+=head2 decode_azure_json
+
+    my $json_obj = decode_azure_json($str);
+
+Helper function to decode json string, retrieved from C<az>, into a json
+object.
+Duo to https://github.com/Azure/azure-cli/issues/9903 we need to strip all
```
Comment (asmorodskyi) [inline lib/publiccloud/azure.pm]:
```suggestion
Due to https://github.com/Azure/azure-cli/issues/9903 we need to strip all
```

**Comment by cfconrad** [comment]
done

**Comment by asmorodskyi** [comment]
Travis is unhappy , there was tidy update maybe you missed it ?

File: lib/publiccloud/azure.pm [hunk 2]
```diff
@@ -270,4 +276,11 @@ sub start_instance
     $instance->public_ip($self->get_ip_from_instance($instance));
 }
 
+
+
+sub create_instances {
```
Comment (jlausuch) [inline lib/publiccloud/azure.pm]: Is this change related to this commit? Why do we need this?

Comment (cfconrad) [reply lib/publiccloud/azure.pm]: ah you are right. It was during development that I changed the terraform apply timeout for Azure to 30min. But I guess it's not needed.  I will remove it!

**Comment by cfconrad** [comment]
done, ready from my side
